tapalcatl_server/handler.go

```diff
@@ -25,23 +28,24 @@ type BufferManager interface {
 	Put(*bytes.Buffer)
 }
 
-// the logState structures and string generation serve to emit a single log entry line
+// the reqState structures and string generation serve to emit a single log entry line
 // a log parser will pick this up and use it to persist metrics
 // the string functions here are specific to the format used and should be updated with care
 
-type logResponseState int32
+type reqResponseState int32
 
 const (
-	ResponseState_Nil logResponseState = iota
+	ResponseState_Nil reqResponseState = iota
```

Comment:
I'm not sure. I think it is considered good practice to have a "zero state" to be the default initialised value, but I'm not sure how that works when that value isn't meaningful. What does it mean if we get to the end of `MetatileHandler`'s `HandlerFunc` and we haven't set the value to something other than `ResponseState_Nil`?

Ideally, we could have the compiler check that no such code path exists. But I can't think of a way to do that, so perhaps we should log `ResponseState_Nil` in the `defer` block as an error?